example/dev-server/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "webapp-example",
+  "version": "1.0.0",
+  "description": "Demo of webpapp webpack plugin",
+  "main": "webpack.config.js",
```

Comment:
It doesn't seem right to me to point webpack config as the main entry point, in any case I believe `main` is optional so could you try removing it?